cubical/machines/abstract_machine.py

```diff
@@ -361,18 +361,20 @@ def __init__(self, machine_cls, grid, double_precision, apply_only, global_optio
             self.init_solutions()
 
         def init_solutions(self):
-            """Internal method. Initializes solution databases. Note that this is reimplemented in JonesChain."""
-            self._init_solutions(self.jones_label, self._make_filename(self.jones_options["load-from"]),
-                                 self.solvable and self._make_filename(self.jones_options["save-to"]),
+            """Internal method, called from constructor. Initializes solution databases.
+            Default behaviour uses the _init_solutions() implementation below.
+            Note that this is reimplemented in JonesChain to collect solution info from chain."""
```

Comment:
If the docstring is not single-line, please use the standard, the quotes should appear before on separate lines.

Reply:
Done.